pkg/load/agents/configAgent.go

```diff
@@ -29,7 +31,7 @@ type IndexFn func(api.ReleaseBuildConfiguration) []string
 
 type configAgent struct {
 	lock         *sync.RWMutex
-	configs      load.FilenameToConfig
+	configs      load.ByOrgRepo
```

Comment:
The `findConfigForJob` function in `webreg` will need to be updated (probably need unit tests for some of the helper functions in that pkg): https://github.com/openshift/ci-tools/blob/master/pkg/webreg/webreg.go#L1820

Reply:
No tests ;)